src/riak_kv_compile_tab.erl

```diff
@@ -165,6 +166,16 @@ get_ddl(BucketType) when is_binary(BucketType) ->
             notfound
     end.
 
+%%
+-spec get_all_compiled_ddls() -> term() | notfound.
+get_all_compiled_ddls() ->
+    case dets:match(?TABLE, {'_','_','$1','_','compiled'}) of
```

Comment:
IIRC you can only have one version of a table at any given time so each table will therefore return only a single DDL record